frontend/src/pages/Deckbuilder.tsx

```diff
@@ -1,19 +1,34 @@
-import styled from "styled-components";
+import styled from '@emotion/styled';
 import {useEffect} from "react";
 import {useStore} from "../hooks/useStore.ts";
 import FetchedCards from "../components/FetchedCards.tsx";
 import SearchForm from "../components/SearchForm.tsx";
+import cardBack from "../assets/cardBack.jpg";
+import CardDetails from "../components/CardDetails.tsx";
 
 export default function Deckbuilder() {
 
     const fetchCards = useStore((state) => state.fetchCards);
+    const selectedCard = useStore((state) => state.selectedCard);
+    const hoverCard = useStore((state) => state.hoverCard);
 
     useEffect(() => {
+        // @ts-ignore
```

Comment:
What's `@ts-ignore`?

Reply:
to supress the typescript error. The function has default values but somehow it's marked as an error